### PR TITLE
Updated ReadMe for Development Instructions

### PR DESCRIPTION
**Pull request by stevegardiner26**
Updated the README for development instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,34 @@
-# Getting Started with Create React App
```
Comment (VivekSreenivasan) [inline README.md:1]: @stevegardiner26 Why was this removed

**Review by VivekSreenivasan — COMMENTED** [review]
This looks good I suppose

File: README.md (hunk 1 above)
Comment (stevegardiner26) [reply README.md:1]: Because it was auto generated for the react part

Comment (VivekSreenivasan) [reply README.md:1]: Interesting

**Comment by stevegardiner26** [comment]
This is now outdated.
